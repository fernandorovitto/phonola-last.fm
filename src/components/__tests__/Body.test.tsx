import 'jest-styled-components';
import Body from '../Body';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Test for Body Component', () => {
  test('Should render the component', () => {
    const tree = renderer.create(<Body />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
