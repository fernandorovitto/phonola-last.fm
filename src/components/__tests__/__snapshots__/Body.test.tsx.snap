// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Body Component Should render the component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-top: 60px;
  width: 100%;
  background-color: #262d31;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  max-width: 738px;
  width: 100%;
}

@media (min-width:992px) {
  .c0 {
    max-width: 962px;
  }
}

@media (min-width:1200px) {
  .c0 {
    max-width: 1140px;
  }
}

<div
  className="c0"
/>
`;
